test/src/test/testtool/Validators/Schema/openapi_post.json

```diff
@@ -5,6 +5,20 @@
         "description":"OpenApi document auto-generated by Frank!Framework for Ibis4Test",
         "version":"0.1.9"
     },
+    "servers":[
+        {
+            "url":"https://loadbalancer.url",
+            "description":"load balancer for LOC server"
+        },
+        {
+            "url":"http://hostname:8085/iaf-test/api",
+            "description":"LOC server hostname"
+        },
+        {
+            "url":"http://localhost:8085/iaf-test/api",
+            "description":"LOC server"
+        }
+    ],
```

Comment:
We should not expose internal urls when not necessary.
So if 'loadBalancer.url' is specified, not other hostname should be returned